src/devices/Gpio/Iot.Device.Gpio.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="../../System.Device.Gpio/**/*.cs" />
```

Comment:
Why do you need to compile the source code of the library as opposed to just reference the library? This is not right since it will expose the surface area of System.Device.Gpio on Iot.Device.Bindings which we definitely don't want as consumers of this library will have conflicts as types will be exposed in two places.